benchmarks/nemo/tracer_advection/scripts/utils.py

```diff
@@ -120,3 +121,68 @@ def insert_explicit_loop_parallelism(
 
             if num_nested_loops > 1:
                 loop.parent.parent.collapse = num_nested_loops
+
+
+def valid_kernel(node):
```

Comment:
This could be used in both acc_mixed_*_trans.py scripts.

Reply:
Using this function (and the insert_explicit_loop) from mixed_trans now.